packages/lodestar/src/sync/reqResp/reqResp.ts

```diff
@@ -218,7 +218,9 @@ export class BeaconReqRespHandler implements IReqRespHandler {
     peerId: PeerId,
     sink: Sink<unknown, unknown>
   ): Promise<void> {
-    this.logger.info(`Received goodbye request from ${peerId.toB58String()}, reason=${request}`);
+    this.logger.info(
+      `Received goodbye request from ${peerId.toB58String()}, reason=${this.config.types.Uint64.toJson(request.body)}`
+    );
```

Comment:
Isn't the logger able to stringify it by itself? This should work or we have to make it work
```ts
this.logger.info("Received goodbye request", {peer: peerId.toB58String(), reason: request.body});
```

Reply:
you're right.  just pushed a change for this.